### DISCUSSION TITLE
Raiments of the Eye Boost on Skill Calculator

### DISCUSSION THREAD

**Discussion by Chad-111** [runelite/runelite#16130] [Ideas]
Adding Raiments of the Eye Boost checkbox on the Skill Calculator plugin would be a nice addition.

**Comment by Nightfirecat** [comment]
Raiments of the Eye do not grant additional experience, so why would that be useful?

**Reply by Chad-111** [reply]
It grants addition runes when the full outfit is worn. Skill Calculator tells you how many actions you need to reach your target.

**Reply by Nightfirecat** [reply]
I don't see how those two facts are related. Crafting multiple runes does not increase your XP rate in runecrafting. You will gain the same amount of experience from crafting 28 essence into air runes regardless of your runecrafting level or number of Raiments of the Eye equipped.
